runtime/parsing.js

```diff
@@ -129,8 +129,8 @@ function caml_parse_engine(tables, env, cmd, arg) {
   var state = env[env_state];
   var errflag = env[env_errflag];
 
-  exit: for (;;) {
-    next: switch (cmd) {
+  the_loop: for (;;) {
```

Comment:
Is that really needed?

```suggestion
  for (;;) {
```

Reply:
nvm